javascript/chronometer.js

```diff
@@ -1,34 +1,44 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0,
+    this.intervalId = null
   }
 
   start(callback) {
-    // ... your code goes here
+    if (!callback){
```

Comment:
Podríamos tener solo una condicional 👀 
```suggestion
 this.intervalId = setInterval( () => {
      this.currentTime++;
      if (callback) callback();
    }, 1000);```